server/src/test/resources/csars/yaml/valid/lamp-csar_noinput/template.yaml

```diff
@@ -0,0 +1,73 @@
+tosca_definitions_version: tosca_simple_yaml_1_0
+description: Template for deploying a LAMP stack.
+metadata:
+  template_name: lamp-stack-template
+  template_author: stupro-toscana
+  template_version: 1.0
+
+
+topology_template:
+  node_templates:
+    my_app:
+      type: tosca.nodes.WebApplication
+      requirements:
+        - host: apache_web_server
+        - database_endpoint: my_db
+      interfaces:
+        Standard:
+          create: my_app/install_php.sh
+          start:
+            implementation:
+              primary: my_app/create_myphpapp.sh
+              dependencies:
+                - myphpapp.php
+                - mysql-credentials.php
+
+    apache_web_server:
+      type: tosca.nodes.WebServer.Apache
+      requirements:
+        - host: server
+
+    my_db:
+      type: tosca.nodes.Database.MySQL
+      properties:
+        name: mydb
+        user: root
+        password: abc
+        port: 3306
+      capabilities:
+        database_endpoint:
+          properties:
+            port: 3306
+      requirements:
+        - host: mysql_dbms
+
+    mysql_dbms:
+      type: tosca.nodes.DBMS.MySQL
+      properties:
+          root_password: abc
+          port: 3306
+        requirements:
+          - host: server
+        interfaces:
+          Standard:
+            configure: mysql_dbms/mysql_dbms_configure.sh
+
+    server:
+      type: tosca.nodes.Compute
+      capabilities:
+        host:
+          properties:
+            num_cpus: 1
+            disk_size: 25 GB
+            mem_size: 2048 MB
+        os:
+          properties:
+            type: linux
+            distribution: ubuntu
+            version: 16.04
+
+    outputs:
+      server_url:
+        description: Concatenated URL with public server address and port
+        value: { concat: [ 'http://', get_attribute: [ server, public_address ], ':', get_attribute: [ server, port ]] }
```

Comment:
We should get rid of the `concat` function in this script aswell

Reply:
Is there an ADR for that explaining this for persons currently not developing CSARs?

Reply:
An adr for what exactly?

Reply:
avoiding concat -- that what your comment said...

Reply:
I simply meant that the concat function is one of these 'nice-to-have' things of the tosca spec. This template however is supposed to not include rather fancy aspects of the specification. This shall enable us to use the template in demo's as soon as possible.
Not sure if we should add an ADR for every feature of the spec that we don't aim to support right now.